tools/ExtensionLibrary/DevHome.ExtensionLibrary/ViewModels/ExtensionLibraryViewModel.cs

```diff
@@ -31,6 +32,13 @@ public partial class ExtensionLibraryViewModel : ObservableObject
     private readonly IExtensionService _extensionService;
     private readonly WindowEx _windowEx;
 
+    // All internal Dev Home extensions that should allow users to be enable/disable them, should add
+    // their class Ids to this set.
+    private readonly HashSet<string> _internalClassIdsToBeShownInExtensionsPage = new()
+    {
+        CommonConstants.HyperVExtensionClassId,

```

Comment:
@kvega005  Once I check this PR in, you'll need to add the SandBox ClassId to this set, so users can enable/disable it in the extensions tab in Dev Home